cfme/infrastructure/provider/openstack_infra.py

```diff
@@ -106,3 +119,27 @@ def node_exist(self, name='my_node'):
         node_uuid = str(nodes_dict[name])
         for db_node in query.all():
             return db_node.hosts.name == str(node_uuid.uuid)
+
+
+@navigator.register(OpenstackInfraProvider, 'All')
```

Comment:
REQUIRED: such destination is already defined in infrastructure/provider/__init__.py. It is common for all infra providers. You don't need to define one more such destination.